pnc-ui/app/keycloak.json

```diff
@@ -0,0 +1,21 @@
+{
+  "realm" : "PNC.REDHAT.COM",
+  "resource" : "pnc-web-local",
+  "realm-public-key" : "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCF6WP9SC6i9ulVqAItdu1qhsN9LoIywlJ3Qw7RUIMmPjKToCnDoq8glkJAkS4I3PbpWaFSBRvSObmAm4XvGv69kl1UPUW76Wgyqvr09fNSNds34MV4NuULE1muHHj3x49RqH4J/UPQvICtE77t+lvjd526uqHPyn+mZlN4B0K6hQIDAQAB",
+  "auth-server-url" : "https://keycloak-pncauth.rhcloud.com/auth",
+  "ssl-required" : "none",
+  "use-resource-role-mappings" : false,
+  "enable-cors" : true,
+  "cors-max-age" : 1000,
```

Comment:
+1
Yes, we do have them in separate wars
we need to have 2 contexts
/pnc-web
/pnc-rest

To be able external apps to access PNC REST backend

On 26.3.2015 11:02, Matej Lazar wrote:
>
> In pnc-ui/app/keycloak.json 
> <https://github.com/project-ncl/pnc/pull/242#discussion_r27199933>:
>
> > @@ -0,0 +1,21 @@
> > +{
> > +  "realm" : "PNC.REDHAT.COM",
> > +  "resource" : "pnc-web-local",
> > +  "realm-public-key" : "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCF6WP9SC6i9ulVqAItdu1qhsN9LoIywlJ3Qw7RUIMmPjKToCnDoq8glkJAkS4I3PbpWaFSBRvSObmAm4XvGv69kl1UPUW76Wgyqvr09fNSNds34MV4NuULE1muHHj3x49RqH4J/UPQvICtE77t+lvjd526uqHPyn+mZlN4B0K6hQIDAQAB",
> > +  "auth-server-url" : "https://keycloak-pncauth.rhcloud.com/auth",
> > +  "ssl-required" : "none",
> > +  "use-resource-role-mappings" : false,
> > +  "enable-cors" : true,
> > +  "cors-max-age" : 1000,
>
> We should keep REST and WEB in different WARs so that you can use 
> (also build) REST without a WEB UI.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/project-ncl/pnc/pull/242/files#r27199933>.
>